src/modules/tools/zprobe/CartGridStrategy.cpp

```diff
@@ -396,11 +470,32 @@ float CartGridStrategy::findBed()
 bool CartGridStrategy::doProbe(Gcode *gc)
 {
     gc->stream->printf("Rectangular Grid Probe...\n");
+
+    if(only_by_two_corners){
+        if(gc->has_letter('X') && gc->has_letter('X') && gc->has_letter('A') && gc->has_letter('B')){
+            x_start = gc->get_value('X'); // override default probe start point, will get saved
+            y_start = gc->get_value('Y'); // override default probe start point, will get saved
+            x_size = gc->get_value('A'); // override default probe width, will get saved
+            y_size = gc->get_value('B'); // override default probe length, will get saved
```

Comment:
“Two corners” not right name for this mode, because it use only one corner and rectangle size. I take it from CAD i'm using. Can you suggest new name? True two corner mode not easy to use...